src/idl/epplex_core.json

```diff
@@ -492,8 +492,5 @@
       "name": "DestroyTimestampNotExceeded",
       "msg": "Destroy timestamp has not been exceeded"
     }
-  ],
-  "metadata": {
-    "address": "epCoD6BqcNinLvKN3KkY55vk4Kxs3W1JTENs1xqWUTg"
-  }
+  ]
```

Comment:
Prolly need to deploy locally - to have the address show up

otherwise can also simply not commit this file

Reply:
turns out that `anchor build` ignores `epplex-core` since there's no diff with the original. 

I've also updated this line https://github.com/epplex-xyz/epPlex/blob/b0dda170848d345b012b4a0508029a09e9778dd2/app/client/idl/epplex_core.json#L497